Dockerfile

```diff
@@ -1,35 +1,28 @@
-FROM debian
-MAINTAINER Andreas Stenius git@astekk.se
+FROM alpine:3.4
+MAINTAINER Marco Wessel <marco@driebit.nl>
 
-ENV DEBIAN_FRONTEND noninteractive
-
-# httpredir.debian.org/debian fails too much, so replace it with a fixed mirror.
-RUN echo \
-   'deb ftp://ftp.nl.debian.org/debian/ jessie main\n \
-    deb ftp://ftp.nl.debian.org/debian/ jessie-updates main\n \
-    deb http://security.debian.org jessie/updates main\n' \
-    > /etc/apt/sources.list
+ADD . /opt/zotonic
+WORKDIR /opt/zotonic
```

Comment:
Better move this below package installation because of Docker’s build layer caching.